powerhome/playbook
# Fixes Typeahead (Rails) setting wrong image attribute in examples

#### Screens

<img width="1281" alt="Screen Shot 2020-10-13 at 1 22 28 PM" src="https://user-images.githubusercontent.com/2293844/95900692-e3cb4f00-0d57-11eb-85b2-42571d967498.png">


#### Breaking Changes

No, this fixes only the docs

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-24

#### How to test this

1) Nitro: Go to `/directory` and search for a user in the filters
2) Check the typeahead docs, ensuring user avatars show up
3) view the inspector console, ensuring no errors occur related to `/placeholder`

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
